src/headers/tomcrypt_pk.h

```diff
@@ -270,6 +277,13 @@ typedef struct {
 
    /** The y co-ordinate of the base point on the curve (hex) */
    const char *Gy;
+
+   /** The co-factor */
+   unsigned long cofactor;
+
+   /** The OID */
+   unsigned long oid[16];
+   unsigned long oidlen;
 } ltc_ecc_set_type;
```

Comment:
is `ltc_ecc_set_type` the right name for this struct? shouldn't it better be called `ltc_ecc_curve`

Reply:
Yes, it would. However, I have not introduced this struct (only added A, cofactor, OID) it already exists and is part of the API. So for backwards compatibility we have to stay with `ltc_ecc_set_type`.

Reply:
we're going to change the API's so why shouldn't we fix the names in the same step?

Reply:
I wanted to keep the changes as small as possible. In fact we are "only" adding to the public API:
* 3 new items into `ltc_ecc_set_type` struct
* 5 completely new functions

Reply:
I'd say with 1.18 "keeping changes small" is finished. We're not going to release a 1.18.1 with these ECC changes, even if the changes don't look that huge we're going to have to bump the ABI again... in the light of the late ABI bump for 1.18 we could (should!?) have included this in 1.18 :/ sorry for that

Reply:
No need to be sorry. Ad ABI versioning I think we can keep it independent on lib versioning and simply bump it when necessary.

Reply:
done (renamed to ltc_ecc_curve)